qt/cmake-project/cmake-project

```diff
@@ -52,17 +58,30 @@ if [ "$QTVERSION" != "none" ]; then
     fi
     QTLIBS="Core Gui"
     QTTARGETS="$QT::Core $QT::Gui"
-    if [ -n "`grep QApplication *.cpp`" ]; then
+    if [ -n "`grep QApplication $SRC`" ]; then
+        echo "QApplication detected"
         QTLIBS="$QTLIBS Widgets"
         QTTARGETS="$QTTARGETS $QT::Widgets"
         AUTOUIC="set(CMAKE_AUTOUIC TRUE)"
     fi
 fi
 
+if [ ! -d "cmake" ]; then
+  mkdir cmake
+fi
+
+SCRIPT=$(dirname -- "$(readlink -f "${BASH_SOURCE}")")
+echo "Running script path $SCRIPT"
```

Comment:
I'd remove this confusing debugging line

Reply:
Fixed